src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
I see. I thought you have a particular concern with .net events. 

What do you think about subscribing to those events from a feature? `Feature.Setup` doesn't really count towards run nor config time. I think it would make sense to allow subscriptions from there.